packages/nextjs/utils/scaffold-eth/getMetadata.ts

```diff
@@ -1,5 +1,10 @@
 import type { Metadata } from "next";
 
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
```

Comment:
Just pushed 95579c29fbfcfe0d725222192347165633eb0775 to have the hardcoded URL here. 

Since a few weeks ago, VERCEL_URL returns the (protected) deployment URL (instead of the live/prod URL) so this doesn't work.